app/views/welcome/events.html.slim

```diff
@@ -15,9 +15,18 @@
         | )
 
     .col.s12
-      ul.collection
+      ul.collection.events-list
+        - current_day = ''
         - cache('all_events') do
-          == render partial: 'shared/event_with_map', collection: @events, as: :event
+          - @events.each do |event|
+            - event_time = Time.at(event['unix_start_time'])
```

Comment:
It might be better to do this kind of grouping / manipulation in controller or in the service class (keep the view clean).